src/Controllers/lessonsPlans.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { lessonPlansScrapper } from '../Services/lessonsPlansScrapper.service';
+import { MajorLessonsPlanObject } from '../Types/majorLessonsPlanObject.type';
+import { Request, RequestHandler, Response } from 'express';
+
+export const getAllLessonsPlans: RequestHandler = async (
+    req: Request,
+    res: Response<Array<MajorLessonsPlanObject>>
```

Comment:
_change type to MajorLessonsPlanObject[]_

Reply:
resolved